public/app.js

```diff
@@ -76,6 +76,11 @@ angular.module('workflow',
                    "|^\\s*" + _wfConfig.incopyExportUrl.match("^.*?:")[0])
         );
 
+        $compileProvider.aHrefSanitizationWhitelist(
```

Comment:
same as below - you could avoid duplicated logic here by having a function which takes the export url as a parameter

Reply:
@philmcmahon Ok, I gave it a shot myself, but need your help with these.

Reply:
Sorry, I will help out tomorrow!